rzbrp100Sup/rzbrp100_channel.template

```diff
@@ -0,0 +1,152 @@
+
+#   Output
+
+record(bi, "$(P)CH$(CHANNEL):OUTPUT")
+{
+    field(DESC, "Channel $(CHANNEL) Output Status")
+    field(DTYP, "stream")
+    field(INP, "@rzbrp100.proto getOutput($(CHANNEL)) $(PORT)")
+    field(SCAN, "2 second")
+    
+    field(ZNAM, "OFF")
+    field(ONAM, "ON")
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:CH$(CHANNEL):OUTPUT")
+    field(SDIS, "$(P)DISABLE")
+    
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+
+    info(alarm, "RZBRP100")
+}
+
+record(bo, "$(P)CH$(CHANNEL):OUTPUT:SP") 
+{
+    field(DESC, "Channel $(CHANNEL) Output setpoint")
+    field(DTYP, "stream")
+    field(OUT,  "@rzbrp100.proto setOutput($(CHANNEL)) $(PORT)")
+
+    field(ZNAM, "OFF")
+    field(ONAM, "ON")
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:CH$(CHANNEL):OUTPUT:SP PP")
+    field(SDIS, "$(P)DISABLE")
+
+    info(INTEREST, "HIGH")
```

Comment:
SP not usually interesting as blocks should almost never be on an SP directly.

Reply:
Have reduced `INTEREST` level to `LOW` for setpoints.